webui: add 'For help debugging' message for FCOS tests

jlebon opened the pull request:
We've added new `coreos.*` tests that now show up in the Bodhi UI for
some packages. These tests are not yet gating, but to be ready for when
they are, let's be nice and link to the #coreos Matrix channel where
packagers can get debugging help like for the other ones.

While we're here, add final periods to each sentence.

jlebon committed:
webui: add 'For help debugging' message for FCOS tests

We've added new `coreos.*` tests that now show up in the Bodhi UI for
some packages. These tests are not yet gating, but to be ready for when
they are, let's be nice and link to the #coreos Matrix channel where
packagers can get debugging help like for the other ones.

While we're here, add final periods to each sentence.

Signed-off-by: Jonathan Lebon <jonathan@jlebon.com>

AdamWill commented:
Hum, we probably want to update the Fedora CI link to https://matrix.to/#/#fedora-ci:fedoraproject.org now too...

jlebon committed:
webui: point 'For help debugging' Fedora CI link to Matrix

Fedora has moved to Matrix by default now so let's update that room. This
matches the other ones.

Signed-off-by: Jonathan Lebon <jonathan@jlebon.com>

jlebon commented:
Sure, tweaked in a separate commit!

mattiaverga committed:
Merge branch 'develop' into pr/coreos-help
